Makefile

```diff
@@ -4,7 +4,7 @@
 PREFIX=/usr/local
 BINDIR=$(PREFIX)/bin
 MANDIR=$(PREFIX)/share/man
-ALL_CFLAGS=$(CFLAGS) -std=c99 -Wall -Wextra -Wpedantic -Wno-unused-parameter
```

Comment:
I just pushed the load average commit, which will now cause a warning when you build with `-DNO_GETLOADAVG`.

I disagree that unused parameters are something that should be warned about. In my opinion, working around the issue with no-op code or comments to silence the warning is worse than just disabling them.

Reply:
I added a commit to fix the warning, I think this is the right way to do it?